src/coreclr/src/System.Private.CoreLib/src/System/Reflection/Assembly.CoreCLR.cs

```diff
@@ -98,5 +98,20 @@ public static Assembly GetCallingAssembly()
 
         [MethodImpl(MethodImplOptions.InternalCall)]
         internal static extern uint GetAssemblyCount();
+
+        internal virtual Attribute? GetCustomAttribute(Type attributeType, bool inherit)
```

Comment:
This virtual is in the shared corelib partition for number of types. Should it be in the shared partition for all of them for consistency? (IL linker should take care of deleting it if it is not used.)